rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.stats/10_basic.yml

```diff
@@ -1,4 +1,16 @@
 ---
+setup:
+  - skip:
+      features: allowed_warnings
+  - do:
+      cluster.health:
+        wait_for_status: green
+  - do:
+      allowed_warnings:
+        - "this request accesses system indices: [.security-7], but in a future major version, direct access to system indices will be prevented by default"
+      indices.delete:
+        index: .security-*
```

Comment:
It's easier to keep this test as is and simply remove .security index before the test.

Reply:
I'm surprised deleting that index doesn't cause any problems.